src/github/client/github-client.ts

```diff
@@ -158,6 +166,17 @@ export default class GitHubClient {
 		});
 	}
 
+	/**
+	 * Get all reviews for a specific pull request.
+	 */
+	public async getPullRequestReviews(owner: string, repo: string, pullNumber: string | number): Promise<AxiosResponse<Octokit.PullsListReviewsResponse>> {
```

Comment:
I will align this method arguments with https://github.com/atlassian/github-for-jira/pull/1117/files

Reply:
I wonder should we make these methods return PullsListReviewsResponse rather than AxiosResponse? (Cuz we handle axios response error in the middleware (if I am not wrong))

Also, half of the methods in this file return data classes like `getCommitsPage` return `getCommitsResponse`

I believe we should incapsulate Axios details inside the class and return the domain model type

Reply:
The Axios fella is wraps it up nicely and gives us the data in a data attribute and a few more attributes like headers, status etc.

The `getCommitsPage` and `getCommitsResponse` are GraphQL calls, but you are probably right that we should try have some consistency!! Calling the GH client and having one return the data and one with an object containing the data might cause some confusion🤔

Reply:
yeah, noticed the inconsistency as well.  I think we need to have a bit of a chat around the Github Client and how we want it to work because I think there are better ways of doing it that is much more consistent.